initial-configuration/jenkins/jenkins-docker/jobs/Configure Outbound Email Settings/config.xml

```diff
@@ -35,17 +35,11 @@
   <concurrentBuild>false</concurrentBuild>
   <builders>
     <hudson.tasks.Shell>
-      <command>export oldpassword=`cat /usr/local/docker-config/wildfly/standalone.xml | grep smtp-server | grep mail-smtp-gmail | grep -oP &apos;password(\S)*&apos; | cut -d \&quot; -f 2`
-		export olduser=`cat /usr/local/docker-config/wildfly/standalone.xml | grep smtp-server | grep mail-smtp-gmail | grep -oP &apos;username(\S)*&apos; | cut -d \&quot; -f 2`
-		export oldfrom=`cat /usr/local/docker-config/wildfly/standalone.xml | grep mail-session | grep &apos;jboss/mail/gmail&apos; | grep -oP &apos;from(\S)*&apos; | cut -d \&quot; -f 2`
-
-sed -i &quot;s/$oldpassword/$OUTBOUND_EMAIL_USER_PASSWORD/g&quot; /usr/local/docker-config/wildfly/standalone.xml
-sed -i &quot;s/$olduser/$OUTBOUND_EMAIL_USER/g&quot; /usr/local/docker-config/wildfly/standalone.xml
-sed -i &quot;s/$oldfrom/$EMAIL_FROM/g&quot; /usr/local/docker-config/wildfly/standalone.xml
-
+      <command>sed -i -E &apos;s#&lt;mail-session name=&quot;([^&quot;]*?)&quot; from=&quot;([^&quot;]*?)&quot; jndi-name=&quot;([^&quot;]*?)&quot; debug=&quot;([^&quot;]*?)&quot;&gt;#&lt;mail-session name=&quot;\1&quot; from=&quot;&apos;&quot;$EMAIL_FROM&quot;&apos;&quot; jndi-name=&quot;\3&quot; debug=&quot;\4&quot;&gt;#g&apos; /usr/local/docker-config/wildfly/standalone.xml
+sed -i -E &apos;s#&lt;smtp-server outbound-socket-binding-ref=&quot;([^&quot;]*?)&quot; tls=&quot;([^&quot;]*?)&quot; username=&quot;([^&quot;]*?)&quot; password=&quot;([^&quot;]*?)&quot; /&gt;#&lt;smtp-server outbound-socket-binding-ref=&quot;\1&quot; tls=&quot;\2&quot; username=&quot;&apos;&quot;$USER&quot;&apos;&quot; password=&quot;&apos;&quot;$PASSWORD&quot;&apos;&quot; /&gt;#g&apos; /usr/local/docker-config/wildfly/standalone.xml
```

Comment:
This looks like it's no longer referencing the variable names passed into this job (lines 16 & 21)

Reply:
Reverted this change.  Ran into an issue running the job multiple times, but at this point I can't remember what the use-case was.  Will revisit if I run across again.